### ISSUE TITLE
FSU reports collection "list view" problem

### ISSUE THREAD

**Issue by lydiam** [FLVC/islandoratheme#41]
CRM Case CAS-59123-H4B7 from FSU reports a problem with collection list view for logged in users:  from within a collection, if you click on "list view" the display is correct, but clicking on "next" results in a display of two lines with the following content"  "<<first < previous 1 2 3 4 5 6 7 8 9 10 ... next> last>>" and no actual records are displayed.

This seems to happen with FireFox, Chrome, and IE browsers.  For comparison I checked UWF's West Florida Photohistory collection and see the same problem (log in with a browser first):  

https://uwf.digital.flvc.org/islandora/object/uwf%3Awfp?page=2&display=list

However, UWF's Lyrasis collection of the same objects displays correctly:

http://ir.uwf.edu/islandora/object/uwf:bell?page=4&display=list

I'm guessing that this is a conflict with something in our theming, but this is just a guess.

Lydia

**Comment by Favenzio** [comment]
I tried to troubleshoot this but every environment seems to display differently. For some reason islandora7d and fsu-test don't even display the paging links (next, last, etc.) when you are logged in and click on 'list view'. I know it is not an islandoratheme problem because I changed the template and the links still did not display. I was able to replicate the problem in the production environment for fau (which does display the links) however, the problem does not appear on fau-test. All this to say, I could not find the source of the problem.
